imagekit/cachefiles/backends.py

```diff
@@ -74,7 +75,7 @@ def set_state(self, file, state):
         if state == CacheFileState.DOES_NOT_EXIST:
             self.cache.set(key, state, self.existence_check_timeout)
         else:
-            self.cache.set(key, state)
+            self.cache.set(key, state, settings.IMAGEKIT_CACHE_TIMEOUT)
```

Comment:
I introduce timeout in order this to be configurable. But in order to not let it to expire I need to change the logic for default value of `IMAGEKIT_CACHE_TIMEOUT` in `imagekit.conf`